mitodl/mitxonline
# Tech Debt: Remove feature flags that are no longer in use

### What are the relevant tickets?
https://github.com/mitodl/hq/issues/4335
https://github.com/mitodl/hq/issues/4064

### Description (What does it do?)
Removes the feature flags from the redesign of the MITx online homepage.  The only feature flags that are still in use are:
```
IGNORE_EDX_FAILURES = "IGNORE_EDX_FAILURES"
SYNC_ON_DASHBOARD_LOAD = "SYNC_ON_DASHBOARD_LOAD"
ENABLE_AUTO_DAILY_FEATURED_ITEMS = "mitxonline-auto-daily-featured-items"
```

### How can this be tested?
- Tests should pass
- The site should load and function correctly
- The redesign should be intact.  
- There should be no calls to posthog outside of the above 3 flags in a dev environment.  You will have to check this in your console OR turn posthog_enabled to true locally.
